packages/flutter_tools/gradle/src/main/groovy/flutter.groovy

```diff
@@ -1178,29 +1180,41 @@ class FlutterPlugin implements Plugin<Project> {
         } // end def addFlutterDeps
 
         if (isFlutterAppProject()) {
+
             project.android.applicationVariants.all { variant ->
                 Task assembleTask = getAssembleTask(variant)
                 if (!shouldConfigureFlutterTask(assembleTask)) {
                   return
                 }
                 Task copyFlutterAssetsTask = addFlutterDeps(variant)
                 copyFlutterAssetsTask.doLast {
+
+                  def content = "";
+                  def outputDir = copyFlutterAssetsTask.destinationDir
+                  def publicKeyFile = new File("${project.rootDir.absolutePath}/../public.pem");
```

Comment:
This feels like it should be specified in shorebird.yaml, no?

Reply:
Like we discussed, changed to read from an env var instead